index.html

```diff
@@ -325,6 +328,44 @@ <h2 class="abtMe_Header">About me
               <ul>
               
     </div>
+    <section class="contact_form">
+        <h1 id="contact_header">I'm always interested in hearing about new projects, so if you'd like to chat 
+            please get in touch.</h1>
+
+            <form action="./contact form" method="post">
```

Comment:
- [ ] As stated in project requirement data should be collected by using the [Formspree service](https://formspree.io/html). So I recommend you to use "action" value  similar to https://formspree.io/f/{form_id}. Because When I submit the Form data displayed Look likes on the browser just like the Image Below, Kindly work on it.

![Capture](https://user-images.githubusercontent.com/93598189/192592683-72dd9b7d-cd06-405e-beb2-720fae90b0f7.PNG)

- And also network tab in your browser dev tools confirm that the form is submitted successfully but without valid input values. So Please make sure that after making adjustment  form is submitted successfully with valid input values.
![Capture](https://user-images.githubusercontent.com/93598189/192593373-029d105c-96ee-489b-b7b7-7f1c88b9cac8.PNG)